invenio_stats/utils.py

```diff
@@ -35,8 +35,14 @@ def get_anonymization_salt(ts):
     return salt
 
 
-def get_bucket_size(client, index, agg_field):
+def get_bucket_size(client, index, agg_field, start_date=None, end_date=None):
     """Function to help us define the size for our search query."""
+    time_range = {}
+    if start_date is not None:
+        time_range['gte'] = start_date.replace(microsecond=0).isoformat()
+    if end_date is not None:
+        time_range['lte'] = end_date.replace(microsecond=0).isoformat()
+
     body = {
         "aggs": {
             "size_count": {
```

Comment:
```suggestion
            "unique_values": {
```

I would rename `size_count` across the function to `unique_value` for better clarity, WDYT?